docs/databases/swift-kuery-orm.html

```diff
@@ -100,8 +100,12 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
     <div id="doc-container" class="docs-item-4 docs-window">
       <main>
         <h1 class="heading-1">Swift Kuery ORM</h1>
-        <p class="block-text">Swift Kuery ORM is an ORM (Object Relational Mapping) library for Swift built on top of <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery">Swift Kuery</a> . Using it allows you to simplify persistence of model objects with your server.</p>
-        <p class="block-text">This guide uses Swift Kuery ORM with <a onclick="setActiveSidebarElementForParent('codable-routing')" href="../routing/codable-routing.html">Codable routes</a> however the Swift Kuery ORM also works with <a onclick="setActiveSidebarElementForParent('raw-routing')" href="../routing/raw-routing.html#">Raw Routing</a>.</p>
+        <p class="block-text"><a href="https://github.com/IBM-Swift/Swift-Kuery-ORM" target="_blank">Swift-Kuery-ORM</a> is an ORM (Object Relational Mapping) library for Swift built on top of <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery">Swift Kuery</a>, using it allows you to simplify the persistence of model objects with your server.</p>
+        <p class="block-text">In this guide we will show you how to save and retrieve a `Codable` Swift model from a database (in this case PostgreSQL) using the ORM.<p/>
+        <p class="block-text">This guide uses the ORM with <a onclick="setActiveSidebarElementForParent('codable-routing')" href="../routing/codable-routing.html">codable routes</a>, however, the ORM also works with Kitura's <a onclick="setActiveSidebarElementForParent('raw-routing')" href="../routing/raw-routing.html#">raw routing</a>.</p>
+        <div class="info">
+          <p>Support for nested `Codable` types (models, structs, arrays etc.) is not currently available in the ORM, it will be added as a V1.0 feature.</p>
```

Comment:
I think I would word this as.... "support for simple nesting will be added prior to releasing V1.0. More complex nesting will be added for V2.0"

Reply:
Requested wording added in latest commit.